digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/SevenRestrictionsLimitationValidatorSpec.scala

```diff
@@ -246,101 +246,13 @@ class SevenRestrictionsLimitationValidatorSpec  extends FunSuite with Matchers {
     }
   }
 
-  test("bogieWeightLimit traffic sign without match asset") {
-    OracleDatabase.withDynTransaction {
-
-      val propTrafficSign = Seq(
-        TrafficSignProperty(0, "trafficSigns_type", "", false, Seq(TextPropertyValue(MaxTonsOnBogieExceeding.OTHvalue.toString))),
-        TrafficSignProperty(1, "trafficSigns_value", "", false, Seq(TextPropertyValue("100"))),
-        TrafficSignProperty(2, "trafficSigns_info", "", false, Seq(TextPropertyValue("200"))))
-
-      val trafficSign = PersistedTrafficSign(1, 1002l, 2, 2, 2, false, 0, 235, propTrafficSign, None, None, None, None, SideCode.TowardsDigitizing.value, None, NormalLinkInterface)
-
-      when(mockRoadLinkService.getRoadLinksWithComplementaryFromVVH(any[BoundingRectangle], any[Set[Int]], any[Boolean])).thenReturn(Seq(roadLink1, roadLink2, roadLink3))
-      when(bogieWeightLimitValidator.dynamicAssetDao.fetchDynamicLinearAssetsByLinkIds(BogieWeightLimit.typeId, Seq(1001l,1002l, 1003l), false)).thenReturn(Seq())
-
-      val result = bogieWeightLimitValidator.assetValidator(trafficSign)
-      result should have size 1
-      result.head.linkId should be (Some(roadLink2.linkId))
-
-      dynamicSession.rollback()
-    }
-  }
-
-  test("bogieWeightLimit traffic sign have a correct asset") {
```

Comment:
Deleted tests needs to be investigated

Reply:
There is a test that checks all asset types have a correct asset related with the all sevenRestrictionsAssets but it was missing the bogieWeightLimit one. So in that test was added the bogieWeightLimit asset type and deleted this test since it was redundant.

```
  test("sevenRestiction traffic sign have a correct asset") {
    sevenRestrictionsAssets.foreach { sevenRestrictionsAsset =>
      massLimitationWithMatchedAsset(sevenRestrictionsAsset)
    }
  }
```